Mathlib/CategoryTheory/MorphismProperty/Representable.lean

```diff
@@ -288,4 +291,87 @@ instance respectsIso : RespectsIso F.relativelyRepresentable :=
 
 end Functor.relativelyRepresentable
 
+namespace MorphismProperty
+
+open Functor.relativelyRepresentable
+
+variable {X Y : D} (P : MorphismProperty C)
+
+/-- Given a morphism property `P` in a category `C`, a functor `F : C ⥤ D` and a morphism
+`f : X ⟶ Y` in `D`. Then `f` satisfies the morphism property `P.relative` with respect to `F` iff:
+* The morphism is representable with respect to `F`
+* For any morphism `g : F.obj a ⟶ Y`, the property `P` holds for any represented pullback of
+  `f` by `g`. -/
+def relative : MorphismProperty D :=
```

Comment:
I feel that the name is a bit too generic but I don't have a better name.

Reply:
I agree that its generic, but I also don't have a better name for it.

Reply:
It's stuck away in an obscure namespace so I think this is fine.